js/checkout.js

```diff
@@ -270,7 +270,7 @@ class Blockonomics {
             container.style.position = null
             parent_container.style.overflow = parent_container.getAttribute('data-bnomics-overflow')
             parent_container.removeAttribute('data-bnomics-overflow')
-        }, 400)
+        }, 300)
```

Comment:
I added a 100ms extra time so that the animation eases out before we remove it, to make it look less abrupt.